chat-commands.js

```diff
@@ -2154,7 +2206,8 @@ exports.commands = {
 		}
 		Users.setOfflineGroup(name, nextGroup);
 
-		this.addModCommand("" + name + " was promoted to " + (Config.groups[nextGroup].name || "regular user") + " by " + user.name + ".");
+		this.addModAction("" + name + " was promoted to " + (Config.groups[nextGroup].name || "regular user") + " by " + user.name + ".");
+		this.modlog(`GLOBAL${(Config.groups[nextGroup].name || "REGULAR").toUpperCase()}`, toId(name));
```

Comment:
``"REGULAR".toUpperCase()`` isn't needed, it's only needed for ``Config.groups[nextGroup].name``